src/routes/api/members.ts

```diff
@@ -0,0 +1,39 @@
+import express from "express";
+
+import { User } from "../../schema";
+import { isAdmin, postParser } from "../middleware";
+
+export let membersRouter = express.Router();
+
+membersRouter.use(isAdmin);
+membersRouter.use(postParser);
+
+membersRouter.post("/", async (request, response) => {
+    try {
+        const emails = (request.body.email as string || "").replace(/\s/g, "").split(",");
+
+        let users = await User.find({ email: { $in: emails } });
+
+        if (users.length != emails.length) {
```

Comment:
You should ignore user emails that are not in Ground Truth.  Otherwise, mass adding a large number of people won't be fun.